lib/input_checkers.rb

```diff
@@ -0,0 +1,26 @@
+require_relative './database_connect'
+require 'bcrypt'
+require 'pg'
+
+class InputCheckers
+
+  extend DatabaseConnect
+  include BCrypt
+
+  def self.unique_input_checker(option, checking_key)
+    result = database_connect.exec("SELECT #{checking_key} FROM users;")
+    result.map do |items|
+      return true if items.values[0] == option
```

Comment:
Again, you could use 'any?' here. The use of 'map' and 'return' feels a bit awkward and its harder to discern intent.